telethon/tl/custom/inlinebuilder.py

```diff
@@ -98,6 +99,13 @@ async def article(
                 The content to be shown for this result.
                 For now it has to be a :tl:`InputWebDocument` if present.
 
+            type (`str`, optional):
```

Comment:
What is this needed for?

Reply:
Using Document and photo, always make bot suffer, from FloodWait, even though it's from web. builder.article is using InputBotInlineResult, which allows InputWebDocument as content. 

So, thought it as shortest and simple way to do, or did it should be named differently/new builder type?

Reply:
Don't `photo` and `document` already work for that, with the web types?

Reply:
They works, for document
Often suffer Suffer FloodWait on UploadMediaRequest, because of which never show results

Reply:
Or now I thinks
What if we use `InputBotInlineResult` in `photo` and `document`too, if its a web link?

Reply:
Does an article of type photo with a photo URL look different from a photo upload locally, or do they look the same?

If they look the same then yes, photo and document should be adjusted to support this. Otherwise we'll need another solution.

Reply:
Yes, they looks same
There is one additional plus point too
We can have title and description while sending Inline Photo

Reply:
article with type `photo`
<br><img src="https://telegra.ph/file/9fc000fd81bcaa729c926.jpg"><hr>

local photo, used with `builder.photo`
<br> <img src="https://telegra.ph/file/1270b3f42267fb93dcbd5.jpg"> <hr>

Tried with `gallery` view too, they looks same